src/LanguageServer/Protocol/Features/Diagnostics/EngineV2/DiagnosticIncrementalAnalyzer.InMemoryStorage.cs

```diff
@@ -1,62 +0,0 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-// See the LICENSE file in the project root for more information.
-
-using System.Collections.Concurrent;
-using System.Collections.Immutable;
-using Microsoft.CodeAnalysis.LanguageServer.Handler.Diagnostics;
-using Roslyn.Utilities;
-
-namespace Microsoft.CodeAnalysis.Diagnostics;
-
-internal partial class DiagnosticAnalyzerService
-{
-    private partial class DiagnosticIncrementalAnalyzer
-    {
-        private static class InMemoryStorage
```

Comment:
all this code away.  this was only used in the ForceAnalyzeProjectAsync codepath to cache its result.  But now we just cache that trivially in a CWT, so this becomes entirely irrelevant.